uopz.h

```diff
@@ -40,7 +40,7 @@ ZEND_END_MODULE_GLOBALS(uopz)
 #define UOPZ(v) (uopz_globals.v)
 #endif
 
-extern zend_class_entry* spl_ce_RuntimeException;
+zend_class_entry* spl_ce_RuntimeException;
```

Comment:
This doesn't look right, because SPL defines the symbol, uopz.c only conditionally defines it. It is extern here for uopz_exception macro ...

Can you try to find another way ?